src/interactionModelGeneration/ControlInteractionModelGenerator.ts

```diff
@@ -248,14 +249,15 @@ function validateTargetSlots(interactionModel: InteractionModelData, targetSlotI
 }
 
 /**
- * Generate the modelData Object by reading the i18n instance.
+ * Produces a lookup table of localized slot and intent definitions used when adding
+ * built-ins to the interaction model.
  *
- * This is the consolidated localization information for the built-ins.  This
- * data is not included in the generated interaction model unless the components
- * are actually used. I.e. this is a map of data from which to extract bits
- * and pieces as needed.
+ * Usage:
+ * - initiative i18next before calling this function.
  */
-export function generateModelData(): ModelData {
+// Exported for internal-use only.
+// TODO: review this convention: use _ prefix for functions that are for internal use only
```

Comment:
maybe remove this TODO since already done ?

Reply:
yes I'm fine with export * if we are careful to put internal functions elsewhere.  That seems like a reasonable thing to enforce & catch in CR via the _ convention.

Reply:
The TODO was to get eyeballs on this new convention.  But I think we just did so I will remove the TODO but also ensure other devs see it.